src/components/Carousel/Carousel.tsx

```diff
@@ -1,43 +1,47 @@
 import { JSX } from '@redneckz/uni-jsx';
+import { useState } from '@redneckz/uni-jsx/lib/hooks';
 import { renderChildren } from '../../BlockRenderer/renderChildren';
+import { BlockDef } from '../../model/ContentPageDef';
 import type { JSXBlock, UniBlockProps } from '../../model/JSXBlock';
 import { BlockWrapper } from '../../ui-kit/BlockWrapper';
-import { ScrollLeftButton, ScrollRightButton } from '../../ui-kit/Button/ArrowButton';
 import { SwipeListContainer } from '../../ui-kit/SwipeListControl/SwipeListContainer';
-import {
-  SwipeListScroll,
-  useSwipeListScroll,
-} from '../../ui-kit/SwipeListControl/useSwipeListScroll';
-import type { CarouselContent } from './CarouselContent';
+import { useSwipeListScroll } from '../../ui-kit/SwipeListControl/useSwipeListScroll';
 import { style } from '../../utils/style';
+import type { CarouselContent } from './CarouselContent';
+import { CarouselScrollButtons } from './CarouselScrollButtons';
 
 export interface CarouselProps extends CarouselContent, UniBlockProps {
   itemCount?: number;
 }
 
 const ITEMS_GAP = 4;
 
-const blurBaseStyle = 'absolute top-0 bottom-0 z-50 w-[84px] pointer-events-none';
-const scrollButtonStyle = 'absolute top-[calc(50%-12px)] z-50';
-
 export const Carousel: JSXBlock<CarouselProps> = JSX<CarouselProps>(
   ({
     className = '',
     itemCount,
     visibleItemCount,
     isBlur = true,
     isCyclic = false,
+    arrowsPadded = true,
     isPadding = true,
     ...rest
   }) => {
     const { blocks: childBlocks } = rest.block || {};
     const count = itemCount || childBlocks?.length || 0;
 
+    const [activeIndex, setActiveIndex] = useState<number>(1);
```

Comment:
странное начальное значение...
константы должны сами за себя говорить.
к примеру, в данном случае

```suggestion
    const [activeIndex, setActiveIndex] = useState(Math.floor(visibleItemCount / 2));
```